src/middlewared/middlewared/plugins/crypto.py

```diff
@@ -1450,7 +1804,8 @@ def __create_acme_certificate(self, job, data):
         cert_info.update({
             'ca_privatekey': signing_cert['privatekey'],
             'ca_certificate': signing_cert['certificate'],
-            'serial': cert_serial
+            'serial': cert_serial,
+            'cert_extensions': data['cert_extensions']
```

Comment:
We pop cert_extensions for some certificates, are we sure cert_extensions key will always be available in this point?

Reply:
Yes for internal certificates they will always be available.